src/server.py

```diff
@@ -57,17 +58,18 @@ def shutdown_server():
 
 @app.route('/ready', methods=['GET'])
 def get_is_ready():
-    return json.dumps('Flask ready')
+    return 'Flask ready'
```

Comment:
Should these tiny functions have docstrings still?

Reply:
Yes, will do.